### DISCUSSION TITLE
Cannot find graphql-dgs-webflux-starter in order to test webflux support 4.0.0-rc.1

### DISCUSSION THREAD

**Discussion by sebastienvermeille** [Netflix/dgs-framework#348] [Q&A]
Hi guys, 
I would like to test the webflux support inside of my app.

According to the release note here:
https://github.com/Netflix/dgs-framework/releases/tag/v4.0.0-rc.1

```
A new starter graphql-dgs-webflux-starter. You use this starter instead of the graphql-dgs-spring-boot-starter if you're using DGS in a Webflux environment.
```

 it seems that there is a spring boot starter for the webflux feature but I cannot find it anywhere here
https://netflixoss.jfrog.io/artifactory/maven-oss-candidates/dgs-framework/

Could someone provide me the dependency references in maven or gradle format? thank you :)

**Comment by ghost** [comment]
implementation(platform("com.netflix.graphql.dgs:graphql-dgs-platform-dependencies:4.0.1"))
    implementation("com.netflix.graphql.dgs:graphql-dgs-webflux-starter")

Example:
https://github.com/Netflix/dgs-examples-webflux/blob/main/build.gradle.kts

**Reply by sebastienvermeille** [reply]
thank you @zanonena will try it this evening!

**Reply by willisjtc** [reply]
@zanonena That example you mention doesn't seem to include any examples of having the data fetchers return a Flux or Mono. Any way that could be added?
